src/main/java/de/tum/in/www1/artemis/service/exam/ExamService.java

```diff
@@ -406,27 +406,25 @@ private ExamBonusCalculator createExamBonusCalculator(Optional<GradingScale> gra
             String bonusGrade = null;
             if (verdict == PlagiarismVerdict.PLAGIARISM) {
                 bonusGrade = GradeStep.PLAGIARISM_GRADE;
-            }
-            else if (bonusExample.bonusGrade() != null) {
+            } else if (bonusExample.bonusGrade() != null) {
                 bonusGrade = bonusExample.bonusGrade().toString();
             }
 
             return new BonusResultDTO(bonusStrategy, bonusFromTitle, bonusExample.studentPointsOfBonusSource(), bonusGrade, bonusExample.finalPoints(), bonusExample.finalGrade(),
-                    verdict, achievedPresentationScore, presentationScoreThreshold);
+                verdict, achievedPresentationScore, presentationScoreThreshold);
         };
     }
 
     private Map<Long, BonusSourceResultDTO> calculateBonusSourceStudentPoints(GradingScale sourceGradingScale, Collection<Long> studentIds) {
         try {
             if (sourceGradingScale.getCourse() != null) {
-                CourseScoresDTO courseScores = courseScoreCalculationService.calculateCourseScores(sourceGradingScale.getCourse().getId(), studentIds);
```

Comment:
This is the only change in this file. The rest is just Intellij reformatting the code. If this is an issue please tell me what our policy is on that.

Reply:
Usually `./gradlew spotlessApply` should trigger automatically as a pre-commit hook to format the Java files accordingly.
The Git hooks (that then also run the client code linting/formatter) should have been installed when running `npm install`.